engines/wage/world.cpp

```diff
@@ -215,6 +215,10 @@ bool World::loadWorld(Common::MacResManager *resMan) {
 		if (res != NULL) {
 			scene->_textBounds = readRect(res);
 			int fontType = res->readUint16BE();
+			// WORKAROUND: Dune Eternity has a weird fontType ID so we override it to the correct one
```

Comment:
Could you please specify what is this "weird type ID"

Reply:
The fontType it reads is 14614 which is way outside of the range of font Types supported. So I hardcoded it to what it should be.